superset/views/lyft.py

```diff
@@ -82,7 +83,10 @@ def sql_json(self):
                 "tab": request.form.get("tab"),
             }
 
-        return self.sql_json_exec(request_json)
+        log_params = {
+            "user_agent": cast(Optional[str], request.headers.get("USER_AGENT"))
+        }
+        return self.sql_json_exec(request_json, log_params)
```

Comment:
You also need to modify `sql_json_exec` to receive the second parameter, no?

Reply:
Yup! That was added in https://github.com/apache/incubator-superset/pull/8826 and lyft-master has been updated with the latest apache.